gizmo/readme.md

```diff
@@ -0,0 +1,24 @@
+Repository for redirecting to the GizMO project
+===================
+Maintainer: Vitalis Wiens (Vitalis.Wiens@gmail.com), Steffen Lohmann (Steffen.Lohmann@iais.fraunhofer.de)
+
+
+This w3id entry is created for the ISWC 2019 Demo: 
+```
+Demonstration of a Customizable Representation Model for 
+Graph-Based Visualizations of Ontologies – GizMO
+
+Abstract. Visualizations can facilitate the development, exploration,
+communication, and sense-making of ontologies. Suitable visualizations,
+however, are highly dependent on individual use cases and targeted user
+groups. In this demo, we present a methodology that enables customiz-
+able definitions for ontology visualizations. We showcase its applicability
+by introducing GizMO, a representation model for graph-based visual-
+izations in the form of node-link diagrams. Additionally, we present two
```

Comment:
Oddly-placed hyphens make reading long words more difficult. I suggest this minor revision --
```
groups. In this demo, we present a methodology that enables customizable
definitions for ontology visualizations. We showcase its applicability by
introducing GizMO, a representation model for graph-based visualizations 
in the form of node-link diagrams. Additionally, we present two 
```

Reply:
Thanks! I have updated the text :)